.idea/.gitignore

```diff
@@ -0,0 +1,3 @@
+# Default ignored files
```

Comment:
Let's add `.idea` to the .gitignore, there is no need to add these to the repo

Reply:
done, thanks!

Reply:
Thank you @cbetta , the files have been removed. Please approve.